tests/test_integrate.cc

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Should this test go into `tests/integrate.cc`?

Reply:
Cleaned up unused code, added unit test for the int1d over Gaussian example in the comments to `tests/integrate.cc? `

Reply:
Yeah, I think that this test here should probably be in `tests/integrate.cc` and not its own file, unless you have a good reason otherwise.

Reply:
Done. The ? was a typo. Copy/paste from your comment.